### PR TITLE
Eclipse OpenJ9 Generative AI (GenAI) Usage Guidelines

### PR DESCRIPTION
**Pull request by 0xdaryl**

### CODE REVIEW

**Comment by 0xdaryl** [comment]
Every active committer should read and comment if you have an opinion. Input from the community is welcome as well. I'd appreciate feedback by Fri. Dec. 13.

If approved, one of @pshipton or @mstoodle should merge please.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -183,6 +188,115 @@ improperly referenced, and the commit is not signed off by the author.
...
+maintain GenAI tags in the code and documentation for as long as possible.
+Existing tags and their boundaries will be maintained on a best-effort basis,
+and they should not be removed simply for the sake of removing them.
+
+However, there are legitimate situations where GenAI tagging may be considered
+for removal, including if the tags:
+
+* interfere with the readability or understandability of the code or
+  documentation,
+* become difficult or unwieldly to maintain (e.g., through code refactoring or
```
Comment (hzongaro) [inline CONTRIBUTING.md:288]: Typo - "unwieldly" should be "unwieldy"

**Review by knn-k — APPROVED** [review]

**Review by hzongaro — APPROVED** [review]

**Comment by 0xdaryl** [comment]
If there are no further comments or objections on this, I'd like to request one of @mstoodle or @pshipton to review/merge today please. Thanks.

**Comment by pshipton** [comment]
I don't think we've seen enough responses from committers to indicate they have looked at it. I asked on Slack, but now I'll ping more committers, skipping those who have seen it in OMR.

@tajila @r30shah @hangshao0 @AdamBrousseau @joransiu @amicic @dmitripivkine @gita-omr @llxia @mpirvu @zl-wang

**Review by zl-wang — APPROVED** [review]

**Review by mpirvu — APPROVED** [review]

**Review by llxia — APPROVED** [review]

**Review by dmitripivkine — APPROVED** [review]

**Review by r30shah — APPROVED** [review]

**Review by amicic — APPROVED** [review]

**Review by AdamBrousseau — APPROVED** [review]

**Review by tajila — APPROVED** [review]

**Review by joransiu — APPROVED** [review]

**Comment by pshipton** [comment]
I think we've got almost everybody we're going to get unless we wait until Jan. I'll merge it this afternoon unless something comes up.

**Review by gita-omr — APPROVED** [review]

**Review by pshipton — APPROVED** [review]

**Pull request merged**